docs/devguide/about.rst

```diff
@@ -0,0 +1,54 @@
+****************
+About This Guide
+****************
+
+..
+    Sect<0.3>
+
+This Developer's Guide corresponds to OpenDDS version |release|.
+This guide is primarily focused on the specifics of using and configuring OpenDDS to build distributed publish-subscribe applications.
+While it does give a general overview of the OMG Data Distribution Service, this guide is not intended to provide comprehensive coverage of the specification.
+The intent of this guide is to help you become proficient with OpenDDS as quickly as possible.
+Readers are encouraged to submit corrections to this guide using a GitHub pull request.
+The source for this guide can be found at :ghfile:`docs/devguide` and :doc:`/internal/docs` contains guidance for editing and building it.
+
+Conventions
+===========
+
+..
+    Sect<0.4.2>
+
+This guide uses the following conventions:
+
+.. list-table::
+   :header-rows: 0
+
+   * - ``Fixed pitch text``
+
+     - Indicates example code or information a user would enter using a keyboard.
+
+   * - *Italic text*
+
+     - Indicates a point of emphasis.
+
+   * - ...
+
+     - An ellipsis indicates a section of omitted text.
+
+``DDS_ROOT``
```

Comment:
RST is so hit and miss about it that I actually didn't think inline markup would work in titles. TIL